Example/Tests/LogstashDestinationTests.swift

```diff
@@ -40,12 +41,32 @@ class LogstashDestinationTests: XCTestCase {
         }
         self.waitForExpectations(timeout: 10.0, handler: nil)
     }
-    
+
+    func testCompletionHandlers() throws {
+        let expect = expectation(description: "Send log expectation")
+        let expect2 = expectation(description: "Send log expectation2")
+        let mockSocket = MockLogstashDestinationSocket(host: "", port: 0, timeout: 5, logActivity: true, allowUntrustedServer: true)
+        let destination = LogstashDestination(socket: mockSocket, logActivity: true)
+        _ = destination.send(.verbose, msg: "{}", thread: "", file: "", function: "", line: 0)
+        _ = destination.send(.debug, msg: "{}", thread: "", file: "", function: "", line: 0)
+        _ = destination.send(.info, msg: "{}", thread: "", file: "", function: "", line: 0)
+        _ = destination.send(.warning, msg: "{}", thread: "", file: "", function: "", line: 0)
+        _ = destination.send(.error, msg: "{}", thread: "", file: "", function: "", line: 0)
+        destination.forceSend { _ in
+            expect.fulfill()
+        }
+        destination.forceSend { _ in
+            expect2.fulfill()
+        }
+        self.waitForExpectations(timeout: 10.0, handler: nil)
```

Comment:
**question, non-blocking**: how come is the timeout so long?

Reply:
You know... Intel... 🤣
Joking apart, it is just the default value that we use in all the unit test in JustLog. I'm not aware of any specific reason for it. It is just a magic number